src/include/migraphx/op/slice.hpp

```diff
@@ -11,6 +11,8 @@
 #include <migraphx/config.hpp>
 #include <cmath>
 #include <utility>
+#include <iostream>
```

Comment:
I don't think this include is needed anymore.

Reply:
Fixed